BotDatabase.py

```diff
@@ -82,47 +105,81 @@ def CleanupBackups(self):
                 
     ### Adding/Updating/Removing Server Entries ###
     def AddBotGuilds(self, ListOwnerAndServerTuples):
-        BotAdditionUpdates = []
         for Entry in ListOwnerAndServerTuples:
-            BotAdditionUpdates.append(Entry + (0,))
-        
-        self.Database.executemany("INSERT INTO servers VALUES(?, ?, ?)", BotAdditionUpdates)
+            stmt = select(Server).where(Server.discord_server_id==Entry.id)
+            result = self.Database.scalars(stmt).first()
+
+            if (result is not None):
+                if (result.deleted_at):
+                    Logger.Log(LogLevel.Warn, f"Bot re-added to previously deleted server")
+                    result.undelete()
+                    self.Database.commit()
+            else:
+                newServer = Server(
+                    discord_server_id = Entry.id,
+                    discord_owner_user_id = Entry.owner_id,
+                ) 
+                self.Database.add(newServer)
         self.Database.commit()
-        Logger.Log(LogLevel.Notice, f"Bot had {len(BotAdditionUpdates)} new server updates")
+        Logger.Log(LogLevel.Notice, f"Bot had {len(ListOwnerAndServerTuples)} new server updates")
         
     def SetNewServerOwner(self, ServerId:int, NewOwnerId:int):
-        ActivationChanges = []
-        ActivationChanges.append({"Id": ServerId, "OwnerId": NewOwnerId})
-        self.Database.executemany("UPDATE servers SET OwnerId=:OwnerId WHERE Id=:Id", ActivationChanges)
+        stmt = select(Server).where(Server.discord_server_id==ServerId)
+        result = self.Database.scalars(stmt).first()
+
+        if (result is None):
+            Logger.Log(LogLevel.Warn, f"Bot had attempted to set new owner on non-existant server ")
+            return False
+        
+        result.discord_owner_user_id = NewOwnerId
+        self.Database.add(result)
         self.Database.commit()
+        return
```

Comment:
Remove extra return here

Reply:
Was unaware of python's automatic explicit return value on end of definition. Will remove